src/zkevm_specs/evm_circuit/execution/create.py

```diff
@@ -130,71 +85,181 @@ def create(instruction: Instruction):
         all_but_one_64th_gas,
     )
 
-    # copy init_code from memory to bytecode
-    copy_rwc_inc, _ = instruction.copy_lookup(
-        instruction.curr.call_id,  # src_id
-        CopyDataTypeTag.Memory,  # src_type
-        instruction.next.code_hash,  # dst_id
-        CopyDataTypeTag.Bytecode,  # dst_type
-        offset,  # src_addr
-        offset + size,  # src_addr_boundary
-        FQ(0),  # dst_addr
-        size,  # length
-        instruction.curr.rw_counter + instruction.rw_counter_offset,
+    ### Do stack depth, nonce and balance pre-check
+    # ErrDepth constraint
+    is_error_depth, _ = instruction.compare(FQ(CALL_CREATE_DEPTH), depth, N_BYTES_STACK)
+    # ErrNonceUintOverflow constraint
+    is_nonce_overflow, _ = instruction.compare(FQ(MAX_U64), nonce, N_BYTES_U64)
```

Comment:
When nonce is added to be `MAX_U64+1`, the `compare` will fail because it's no longer in 8-byte, so we should check `nonce_prev` is less than `MAX_U64` here.

```suggestion
    # ErrNonceUintOverflow constraint
    is_nonce_in_range, _ = instruction.compare(nonce_prev, FQ(MAX_U64), N_BYTES_U64)
```

Reply:
fixed in [048ab74](https://github.com/privacy-scaling-explorations/zkevm-specs/pull/416/commits/048ab74e8c19e9d3d997c416f42221c68afe31a1)